config/jobs/kubernetes/wg-k8s-infra/trusted/releng/releng-trusted.yaml

```diff
@@ -63,7 +63,7 @@ periodics:
   spec:
     serviceAccountName: k8s-infra-promoter
     containers:
-    - image: k8s.gcr.io/artifact-promoter/kpromo:v0.2.4-1
+    - image: k8s.gcr.io/artifact-promoter/kpromo:v3.2.0-1
       command:
       - /kpromo
       args:
```

Comment:
I think it now needs `--confirm` and drop `--dry-run` bellow?

Reply:
ref: https://github.com/kubernetes/test-infra/pull/23591#issuecomment-919277894